### PR TITLE
LPS-200103 | Add new property `pathParameterDescription`

### PR DESCRIPTION
**Pull request by jaimelr10**
**Description:** Add the property `pathParameterDescription` to the API Endpoint `objectFields` to contain the value of the field Path Parameter Description in the API Builder
image

**Jira Ticket**: https://liferay.atlassian.net/browse/LPS-200103

### CODE REVIEW

**Review by 4lejandrito — APPROVED** [review]

**Review by 4lejandrito — COMMENTED** [review]
See my latest comment. If this cannot be deployed on an existing db with api builder object definitions, then we have a problem because API Builder has already been released as beta.

File: modules/apps/headless/headless-builder/headless-builder-impl/src/main/java/com/liferay/headless/builder/internal/model/listener/APIEndpointRelevantObjectEntryModelListener.java [hunk 1]
```diff
@@ -148,9 +148,22 @@ private void _validate(ObjectEntry objectEntry) {
...
 					APIApplication.Endpoint.RetrieveType.SINGLE_ELEMENT)) {
 
+				String pathParameter = (String) values.get("pathParameter");
+
 				_validateSingleElementPath(
-					objectEntry, (String)values.get("pathParameter"),
+					objectEntry, pathParameter,
 					pathString, responseAPISchemaId);
+
+				String pathParameterDescription =
```
Comment (sergiojimcos) [inline modules/apps/headless/headless-builder/headless-builder-impl/src/main/java/com/liferay/headless/builder/internal/model/listener/APIEndpointRelevantObjectEntryModelListener.java:157]: Inline

File: modules/apps/headless/headless-builder/headless-builder-impl/src/main/java/com/liferay/headless/builder/internal/model/listener/APIEndpointRelevantObjectEntryModelListener.java [hunk 2]
```diff
@@ -148,9 +148,22 @@ private void _validate(ObjectEntry objectEntry) {
...
+					(String)values.get("pathParameterDescription");
+
+				if (Validator.isNull(pathParameter) && Validator.isNotNull(pathParameterDescription)) {
```
Comment (sergiojimcos) [inline modules/apps/headless/headless-builder/headless-builder-impl/src/main/java/com/liferay/headless/builder/internal/model/listener/APIEndpointRelevantObjectEntryModelListener.java:160]: I think it needs a SF

**Review by sergiojimcos — CHANGES_REQUESTED** [review]
Just small comments

**Comment by 4lejandrito** [comment]
@jaimelr10 if the field is added to an existing DB, I think this is good to go to Brian.

**Review by sergiojimcos — APPROVED** [review]
LGMT!!

**Comment by magjed4289** [comment]
test failures are not related. OK to manually forward ✅

**Comment by jaimelr10** [comment]
ci:forward:force

**Comment by jaimelr10** [comment]
ci:forward:force

**Pull request closed without merging**